src/Components/usersList.js

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import {Checkbox} from "@material-ui/core";
+import styled from 'styled-components';
+
+const TableHeadCellStyled = styled(TableCell)`
+        background: #CED2AA;  
+        font-size : large;
+`;
+
+const TableBodyCellStyled = styled(TableCell)`
+        font-size: 14px;
+`;
+
+const TableStyled = styled(Table)`
+    min-width: 100px;
+    margin-top: 10px;
+`;
+
+export default function UsersList() {
+
+    const [users] = useState([{
+        firstName: null,
+        lastName: null,
+        email: null,
+        dateCreated: null,
+        dateModified: null,
+    }]);
```

Comment:
The implementation before this was fine, just make the two following changes:
`const [users] = useState(null);`
`{users && users.map((user, index) => (`